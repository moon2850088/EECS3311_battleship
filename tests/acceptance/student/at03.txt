--easy no win
debug_test(easy)
fire([B,2])
fire([A,4])
fire([D,1])
fire([B,3])
fire([C,3])
fire([D,2])
fire([D,3])
fire([D,4])
bomb([D,4],[C,4])
bomb([A,2],[A,3])
