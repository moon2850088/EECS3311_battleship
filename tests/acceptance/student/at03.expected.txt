  state 0 OK -> Start a new game
->debug_test(easy)
  state 1 OK -> Fire Away!
     1  2  3  4
  A  _  _  _  _
  B  _  _  _  v
  C  _  _  h  v
  D  _  _  _  _
  Current Game (debug): 1
  Shots: 0/8
  Bombs: 0/2
  Score: 0/3 (Total: 0/3)
  Ships: 0/2
    2x1: [B, 4]->v;[C, 4]->v
    1x1: [C, 3]->h
->fire([B, 2])
  state 2 OK -> Miss! Keep Firing!
     1  2  3  4
  A  _  _  _  _
  B  _  O  _  v
  C  _  _  h  v
  D  _  _  _  _
  Current Game (debug): 1
  Shots: 1/8
  Bombs: 0/2
  Score: 0/3 (Total: 0/3)
  Ships: 0/2
    2x1: [B, 4]->v;[C, 4]->v
    1x1: [C, 3]->h
->fire([A, 4])
  state 3 OK -> Miss! Keep Firing!
     1  2  3  4
  A  _  _  _  O
  B  _  O  _  v
  C  _  _  h  v
  D  _  _  _  _
  Current Game (debug): 1
  Shots: 2/8
  Bombs: 0/2
  Score: 0/3 (Total: 0/3)
  Ships: 0/2
    2x1: [B, 4]->v;[C, 4]->v
    1x1: [C, 3]->h
->fire([D, 1])
  state 4 OK -> Miss! Keep Firing!
     1  2  3  4
  A  _  _  _  O
  B  _  O  _  v
  C  _  _  h  v
  D  O  _  _  _
  Current Game (debug): 1
  Shots: 3/8
  Bombs: 0/2
  Score: 0/3 (Total: 0/3)
  Ships: 0/2
    2x1: [B, 4]->v;[C, 4]->v
    1x1: [C, 3]->h
->fire([B, 3])
  state 5 OK -> Miss! Keep Firing!
     1  2  3  4
  A  _  _  _  O
  B  _  O  O  v
  C  _  _  h  v
  D  O  _  _  _
  Current Game (debug): 1
  Shots: 4/8
  Bombs: 0/2
  Score: 0/3 (Total: 0/3)
  Ships: 0/2
    2x1: [B, 4]->v;[C, 4]->v
    1x1: [C, 3]->h
->fire([C, 3])
  state 6 OK -> 1x1 ship sunk! Keep Firing!
     1  2  3  4
  A  _  _  _  O
  B  _  O  O  v
  C  _  _  X  v
  D  O  _  _  _
  Current Game (debug): 1
  Shots: 5/8
  Bombs: 0/2
  Score: 1/3 (Total: 1/3)
  Ships: 1/2
    2x1: [B, 4]->v;[C, 4]->v
    1x1: [C, 3]->X
->fire([D, 2])
  state 7 OK -> Miss! Keep Firing!
     1  2  3  4
  A  _  _  _  O
  B  _  O  O  v
  C  _  _  X  v
  D  O  O  _  _
  Current Game (debug): 1
  Shots: 6/8
  Bombs: 0/2
  Score: 1/3 (Total: 1/3)
  Ships: 1/2
    2x1: [B, 4]->v;[C, 4]->v
    1x1: [C, 3]->X
->fire([D, 3])
  state 8 OK -> Miss! Keep Firing!
     1  2  3  4
  A  _  _  _  O
  B  _  O  O  v
  C  _  _  X  v
  D  O  O  O  _
  Current Game (debug): 1
  Shots: 7/8
  Bombs: 0/2
  Score: 1/3 (Total: 1/3)
  Ships: 1/2
    2x1: [B, 4]->v;[C, 4]->v
    1x1: [C, 3]->X
->fire([D, 4])
  state 9 OK -> Miss! Keep Firing!
     1  2  3  4
  A  _  _  _  O
  B  _  O  O  v
  C  _  _  X  v
  D  O  O  O  O
  Current Game (debug): 1
  Shots: 8/8
  Bombs: 0/2
  Score: 1/3 (Total: 1/3)
  Ships: 1/2
    2x1: [B, 4]->v;[C, 4]->v
    1x1: [C, 3]->X
->bomb([D, 4],[C, 4])
  state 10 Already fired there -> Keep Firing!
     1  2  3  4
  A  _  _  _  O
  B  _  O  O  v
  C  _  _  X  v
  D  O  O  O  O
  Current Game (debug): 1
  Shots: 8/8
  Bombs: 0/2
  Score: 1/3 (Total: 1/3)
  Ships: 1/2
    2x1: [B, 4]->v;[C, 4]->v
    1x1: [C, 3]->X
->bomb([A, 2],[A, 3])
  state 11 OK -> Miss! Keep Firing!
     1  2  3  4
  A  _  O  O  O
  B  _  O  O  v
  C  _  _  X  v
  D  O  O  O  O
  Current Game (debug): 1
  Shots: 8/8
  Bombs: 1/2
  Score: 1/3 (Total: 1/3)
  Ships: 1/2
    2x1: [B, 4]->v;[C, 4]->v
    1x1: [C, 3]->X
